test/common.bash

```diff
@@ -74,3 +74,25 @@ removeAll() {
 removeUniqueTag() {
     run bash -c "linode-cli tags delete $uniqueTag"
 }
+
+createLinodeAndWait() {
+    local default_plan=$(linode-cli linodes types --text --no-headers --format="id" | xargs | awk '{ print $1 }')
+    local linode_type=${1:-$default_plan}
+    run bash -c "linode-cli linodes create --type=$linode_type --region us-east --image=$test_image --root_pass=$random_pass"
+    assert_success
+
+    local linode_id=$(linode-cli linodes list --format id --text --no-header | head -n 1)
+
+    SECONDS=0
+    until [ $(linode-cli linodes view $linode_id --format="status" --text --no-headers) = "running" ]; do
+        echo 'still provisioning'
+        sleep 3 # Wait 3 seconds between requests
+        if [[ "$SECONDS" -eq 180 ]];
```

Comment:
this might be me being naive, but is anything incrementing SECONDS here?

Reply:
Type `echo $SECONDS` in your shell, then type `SECONDS=0` then `echo $SECONDS` again 😉

Reply:
TIL